src/master/operations/backwards-compatibility.md

```diff
@@ -12,27 +12,35 @@ needed an updated method of proof calculation in order to verify newly added dat
 In some cases it is very hard or impossible to update the verification code on the client side.
 If this is the case, immudb offers a way to disable metadata to maintain compatibility with older clients.
 
+</WrappedSection>
+
+<WrappedSection>
+
 #### Enabling the 1.1 proof compatibility mode
 
 *Note: backwards compatibility mode is currently not available for the `detaultdb` database.*
 
 When creating new database, the mode can be specified with:
 
-```sh
-immuadmin database create <db-name> --write-tx-header-version 0
+```bash
+$ ./immuadmin database create <db-name> --write-tx-header-version 0
```

Comment:
For future improvement:
https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md014

Reply:
yes, need to revisit this, but later in separate PR :)